app/models/responsible_body.rb

```diff
@@ -35,7 +34,7 @@ class ResponsibleBody < ApplicationRecord
   after_update :maybe_generate_user_changes
 
   def self.chosen_who_will_order
```

Comment:
This method name is also misleading. It should be clear that this is only a default in absence of a specific setting for the school.

In fact this method only seems to be used in the performance dashboard and the same distinction could be made there:
https://github.com/DFE-Digital/get-help-with-tech/blob/cde94d3932fabaff56896480b73fc7f83f88b0b8/app/models/support/service_performance.rb#L384-L388

Reply:
I have searched the code and this is not used anymore. Deleted!